pgscatalog_utils/match/match_variants.py

```diff
@@ -65,18 +65,37 @@ def match_variants():
                 raise Exception
 
         dataset = args.dataset.replace('_', '-')  # underscores are delimiters in pgs catalog calculator
-        valid_matches, filter_summary = filter_scores(scorefile=scorefile, matches=matches, dataset=dataset,
-                                                      min_overlap=args.min_overlap)
 
-        if valid_matches.fetch().is_empty():  # this can happen if args.min_overlap = 0
-            logger.error("Error: no target variants match any variants in scoring files")
-            raise Exception
+        if args.only_match:
+            fout: str = f"{dataset}_{args.chrom}_matches.ipc.zst"
+            logger.debug(f"--only_match set, writing out match candidates {fout} and exiting")
+            matches.collect().write_ipc(fout, compression="zstd")
+            logger.debug("Intermediate files can be processed with combine_matches")
+            sys.exit(0)
+        else:
+            logger.debug("Picking best match candidates and making scoring files")
```

Comment:
I don't think this is true. It's actually already selected the best match candidates because the labelling is applied after the matching. Overall I wonder if this is a bad strategy? Like say someone gave 2 pvars for chromosome 1 with overlapping variants.... what would happen? I wonder maybe if we should add a QC check to make sure there are no duplicate variant_ids across pvars?